orderer/consensus/etcdraft/consenter_test.go

```diff
@@ -151,10 +157,35 @@ var _ = Describe("Consenter", func() {
 		)
 
 		BeforeEach(func() {
-			tlsCA, _ := tlsgen.NewCA()
+			tlsCA, err := tlsgen.NewCA()
+			Expect(err).NotTo(HaveOccurred())
 			confAppRaft := genesisconfig.Load(genesisconfig.SampleDevModeEtcdRaftProfile, configtest.GetDevConfigDir())
 			confAppRaft.Consortiums = nil
 			confAppRaft.Consortium = ""
+
+			//IsChannelMember verifies config meta along with it's tls certs of consenters.
+			//So when we add new conseter with tls certs, they must be signed by any msp from orderer config.
+			//Consenters in this test have certs generated by tlsCA which is not part of existing config.
+			//Adding tls ca root cert to an existing ordering org's MSP definition. Don't want to spoil sampleconfig, copying it to some tmp dir.
+			mspDir, err := ioutil.TempDir("", "msp-")
+			Expect(err).NotTo(HaveOccurred())
+			Expect(confAppRaft.Orderer).NotTo(BeNil())
+			Expect(confAppRaft.Orderer.Organizations).To(HaveLen(1))
+
+			err = testutil.CopyDir(confAppRaft.Orderer.Organizations[0].MSPDir, mspDir, true)
+			Expect(err).NotTo(HaveOccurred())
+			//confAppRaft.Orderer.Organizations[0].MSPDir = mspDir
+			//confAppRaft.Orderer.Organizations[0].ID = fmt.Sprintf("SampleMSP-%d", time.Now().UnixNano())
```

Comment:
Nit: remove?

Reply:
I'm in a process of debugging it 
https://chat.hyperledger.org/channel/fabric-orderer?msg=vjDwFDFtGvhWHKzKB here is the problem. 
will cleanup & remove ofc